packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -151,25 +152,16 @@ function FastBridgesTable() {
 }
 
 export function WithdrawalConfirmationDialog(props: UseDialogProps) {
-  const { l1 } = useNetworksAndSigners()
+  const { l1, l2 } = useNetworksAndSigners()
 
   const [checkbox1Checked, setCheckbox1Checked] = useState(false)
   const [checkbox2Checked, setCheckbox2Checked] = useState(false)
 
   const bothCheckboxesChecked = checkbox1Checked && checkbox2Checked
 
-  const isTestnet = useMemo(() => {
-    if (typeof l1.network === 'undefined') {
-      return true
-    }
-
-    return l1.network.chainID !== 1
-  }, [l1.network])
-
-  const confirmationPeriod = useMemo(
-    () => (isTestnet ? '~1 day' : '~8 days'),
-    [isTestnet]
-  )
+  const { isMainnet } = isNetwork(l1.network)
+  const confirmationPeriod = isMainnet ? '~8 days' : '~1 day'
```

Comment:
We should make this dynamic at some point probably

Reply:
```ts
const timeInSecs =  l2.network.confirmPeriodBlocks / l1.network.blockTime + expectedOverhead
```